controllers/RecruiterConroller.js

```diff
@@ -0,0 +1,114 @@
+  body("userRole", "user Role must not be empty.").notEmpty(),
+  body("mobileNo", "mobileNo must be 10 digits.").isLength({
+    min: 10,
+    max: 10,
+  }),
+  body("locations", "locations must not be empty.").notEmpty().isArray(),
+  body("website", "website must not be empty.").notEmpty(),
+  body("foundationYear", "foundation Year must not be empty.").isNumeric(),
+  body("noOfEmployees", "no Of Employees must not be empty.").isNumeric(),
+  body("aboutCompany", "about must not be empty.").notEmpty(),
+  body("companyPhotos", "company Photos must not be empty.").isArray({
+    max: 5,
+  }),
+  body("companyLogo", "companyLogo must be empty.").notEmpty(),
+  (req, res) => {
+    try {
+      const user = req.user;
+
+      if (user.userType !== constants.userTypes.RECRUITER)
+        return apiResponse.unauthorizedResponse(res, "Invalid Permission.");
+
+      const errors = validationResult(req);
+      if (!errors.isEmpty()) {
+        return apiResponse.validationErrorWithData(
+          res,
+          "Validation Error.",
+          errors.array()
+        );
+      } else {
+        const payload = {
+          companyName: req.body.companyName,
+          companyLogo: req.body.companyLogo,
+          userRole: req.body.userRole,
+          mobileNo: req.body.mobileNo,
+          locations: req.body.locations,
+          website: req.body.website,
+          foundationYear: req.body.foundationYear,
+          noOfEmployees: req.body.noOfEmployees,
+          aboutCompany: req.body.aboutCompany,
+          companyPhotos: req.body.companyPhotos,
+        }
+
+        if(req.body.linkedinProfile)
+          payload.linkedinProfile = req.body.linkedinProfile
+        if(req.body.twitterProfile)
+          payload.twitterProfile =  req.body.twitterProfile
+        if(req.body.facebookProfile)
+          payload.facebookProfile = req.body.facebookProfile
+
+        const recruiter = new RecruiterModel(payload);
+        
+        recruiter.save((error, recruiter) => {
+          if (error) {
+            return apiResponse.ErrorResponse(
+              res,
+              "Error Saving Profile",
+              error
+            );
+          }
+
+          UserModel.findOneAndUpdate(
+            { _id: user.id },
+            { $set: { recruiter: recruiter._id, isOnboardingCompleted: true } },
+            {
+              new: true,
+              projection: {
+                name: 1,
+                email: 1,
+                userType: 1,
+                isOnboardingCompleted: 1,
+                recruiter: 1,
+              },
+            },
+            (error, user) => {
+              if (error) {
+                return apiResponse.ErrorResponse(
+                  res,
+                  "Error Saving Profile",
+                  error
+                );
+              }
+
+              UserModel.populate(
+                user,
+                { path: "recruiter", model: "Recruiter" },
+                (err, user) => {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 8 tabs but found 16 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
								(err, user) => {
```